testament/testament.nim

```diff
@@ -107,7 +107,8 @@ proc isSuccess(input: string): bool =
   # that may appear in user config (eg: `--filenames`).
   # Passing `XDG_CONFIG_HOME= testament args...` can be used to ignore user config
   # stored in XDG_CONFIG_HOME, refs https://wiki.archlinux.org/index.php/XDG_Base_Directory
-  input.startsWith("Hint: ") and input.endsWith("[SuccessX]")
+  input.startsWith("Hint: ") and
```

Comment:
why is this diff needed?
`callNimCompiler` checks each line for `isSuccess`

Reply:
A [BuildMode] ending line is a new kind of [SuccessX] line is the simple way to say it.

Reply:
I actually mentioned this & alternatives in my 2nd comment on this thread.